WordPressKitTests/JetpackProxyServiceRemoteTests.swift

```diff
@@ -0,0 +1,122 @@
+import XCTest
+@testable import WordPressKit
+
+class JetpackProxyServiceRemoteTests: XCTestCase {
+    let timeout: TimeInterval = 1.0
+    let api = MockWordPressComRestApi()
+    let siteID = 1001
+
+    private var remote: JetpackProxyServiceRemote {
+        .init(wordPressComRestApi: api)
+    }
+
+    // MARK: - Tests
+
+    func testProxyRequestEndpointIsCorrect() {
```

Comment:
I would say we can drop "isCorrect" from each test as any test would check for correctness of the suite it targets.

Reply:
I personally dislike vague test names and prefer them to clearly communicate the intention / what is being tested & expected. But, fair enough, the `IsCorrect` bit doesn't help and is a bit redundant 😅 . This is addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/576/commits/1dc66a6fe74c2249f3ca5f253f4e8c72eccf6dd3.